paasta_itests/setup_marathon_job.feature

```diff
@@ -18,6 +18,19 @@ Feature: setup_marathon_job can create a "complete" app
       And we run setup_marathon_job until it has 5 task(s)
      Then we should see the number of instances become 5
 
+  Scenario: marathon apps can be scaled up with at-risk hosts
+    Given a working paasta cluster
+      And a new healthy app to be deployed, with bounce strategy "crossover" and drain method "noop" and constraints [["hostname", "UNIQUE"]]
+      And an old app to be destroyed with constraints [["hostname", "UNIQUE"]]
+    When there are 2 old healthy tasks
+     And we mark the host "mesosslave" as at-risk
+     And setup_service is initiated
+    When there are 2 new healthy tasks
+     And setup_service is initiated
+     And we wait a bit for the old app to disappear
+    Then the old app should be gone
+     And the tasks on the host "mesosslave" should be drained
```

Comment:
I'd like to see another integration test that checks that this all works with only a new app.